scripts/pages.mjs

```diff
@@ -43,6 +43,69 @@ class ClassPages extends Application {
     html[0].querySelectorAll(".class-nav img[data-action]").forEach(n => {
       n.addEventListener("click", this._onClickClass.bind(this));
     });
+    this.navScroller(html);
```

Comment:
Generally we try to bind all event listeners in the `activateListeners` method rather than calling other internal functions.